### PR TITLE
Качальский Илья 2536

### PR DESCRIPTION
**Pull request by IlyaHalsky**
-Changed colors
-Deleted anything that "Сборщик мусора" can collect.
-Added Bitmap ( Seems working a bit faster)
-Scaled picture to right conditions
-FPS is now around 15
-The greatest improvment: Icons!!!

### CODE REVIEW

**Comment by thevery** [comment]
ok = 8
~12 FPS

**Comment by thevery** [comment]
a lot of useless files was commited, aargh!

File: app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java [hunk 1]
```diff
@@ -10,15 +9,18 @@
...
-    int [][] field = null;
-    int width = 0;
-    int height = 0;
-    int scale = 4;
-    final int MAX_COLOR = 10;
-    int[] palette = {0xFFFF0000, 0xFF800000, 0xFF808000, 0xFF008000, 0xFF00FF00, 0xFF008080, 0xFF0000FF, 0xFF000080, 0xFF800080, 0xFFFFFFFF};
+    int[][] field = null;
+    int[][] field2 = null;
+    int[] bits = null;
+    int width = 240;
```
Comment (thevery) [inline app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java:18]: `public static final`

File: app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java [hunk 2]
```diff
@@ -10,15 +9,18 @@
...
+    int height = 320;
+    float scale_W = 1;
```
Comment (thevery) [inline app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java:20]: bad naming

File: app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java [hunk 3]
```diff
@@ -53,61 +56,66 @@ public void run() {
...
             }
         }
     }
 
     @Override
     public void onSizeChanged(int w, int h, int oldW, int oldH) {
-        width = w/scale;
-        height = h/scale;
+        /*width = w/scale;
+        height = h/scale;*/
```
Comment (thevery) [inline app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java:68]: commented code should be removed

File: app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java [hunk 4]
```diff
@@ -53,61 +56,66 @@ public void run() {
...
+        scale_H = (float) w / width;
+        scale_W = (float) h / height;
+        Log.i("TIME", "Circle: " + scale_W);
+        Log.i("TIME", "Circle: " + scale_H);
```
Comment (thevery) [inline app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java:72]: bad logging

**Comment by thevery** [comment]
`ic_launcher2.png` is unused unless you have changed manifest

**Comment by IlyaHalsky** [comment]
Its just copied icon, the brand new one is useв just by replacing.

**Pull request closed without merging**